paseos/attitude/disturbance_torques_utils.py

```diff
@@ -0,0 +1,262 @@
+"""this file contains functions that return attitude disturbance torque vectors expressed in the actor body frame"""
+
+# OUTPUT NUMPY ARRAYS
+# OUTPUT IN BODY FRAME
+import numpy as np
+from ..utils.reference_frame_transfer import (
+    rpy_to_body,
+    eci_to_rpy,
+    compute_transformation_matrix_rpy_body,
+)
+
+
+def compute_aerodynamic_torque(r, v, mesh, actor_attitude_in_rad, current_spacecraft_temperature_K):
+    """Calculates the aerodynamic torque on the satellite.
+    The model used is taken from "Roto-Translational Spacecraft Formation Control Using Aerodynamic Forces"; Ran. S,
+    Jihe W., et al.; 2017. The mass density of the atmosphere is calculated from the best linear fit of the data
+    extracted from "Thermospheric mass density: A review"; Emmert J.T.; 2015. This function only works for Earth
+    centered orbits and satellites defined by a cuboid mesh as in the geometrical model.
+
+    Future addition to the model can be the temperature of the spacecraft as an input parameter. At the moment the
+    temperature of the spacecraft is assumed in the model. The sensitivity of the torque with respect to the
+    temperature of the spacecraft and of the gas is low.
+
+    Args:
+        r (np.array): distance from the satellite and the Earth's center of mass [km].
+        v (np.array): velocity of the satellite in ECI reference frame [m/s].
+        mesh (trimesh): mesh of the satellite from the geometric model.
+        actor_attitude_in_rad (np.array): spacecraft actor in rad.
+        current_spacecraft_temperature_K (float): current temperature in Kelvin.
+     Returns:
+         T (np.array): torque vector in the spacecraft body frame.
+    """
+    # Constants for aerodynamic coefficients calculation
+    temperature_gas = 1000  # [K]
+    R = 8.314462  # Universal Gas Constant, [J/(K mol)]
+    altitude = np.linalg.norm(r) - 6371  # [km]
```

Comment:
hard-coded central body radius, can be different and in other places another one is used

Reply:
also wrong unit, paseos uses m

Reply:
I will fix this, but I have already opened an issue to test the aerodynamic model separately. It was delivered and tested as a standalone code. So its integrability in PASEOS is really questionable. I am considering adding a not implemented issue, so we can check this model more without blocking the use of attitude modelling.